spec/requests/api/v2/allocations_spec.rb

```diff
@@ -58,6 +58,29 @@
     end
   end
 
+  describe "DELETE" do
```

Comment:
Same here?

Reply:
I'm not quite sure what the right way to phrase it is - it's a DELETE method in the API call, but it's a destroy method in the allocations controller!